routing/road_graph.cpp

```diff
@@ -122,8 +117,8 @@ bool Edge::operator<(Edge const & r) const
     return m_featureId < r.m_featureId;
   if (m_forward != r.m_forward)
     return (m_forward == false);
-  if (m_partOfReal != r.m_partOfReal)
-    return (m_partOfReal == false);
+  if (m_type != r.m_type)
+    return m_type < r.m_type;
```

Comment:
Let's keep the order of params in operator< as well.

Reply:
As a variant we may place |m_type| after |m_forward|. So |m_type| will take place of |m_partOfReal|.

Reply:
Done.